.github/workflows/ci.yaml

```diff
@@ -64,9 +65,22 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Setup operator environment
+      - name: Setup k8s controller
         # TODO: Replace with custom image on self-hosted runner
-        uses: charmed-kubernetes/actions-operator@main
+        uses: sed-i/actions-operator@patch-1
+        with:
+          juju-channel: 2.9/stable
```

Comment:
it might be really nice to make this matches when you bootstrap the lxd controller as well

Reply:
Good point. Added.